src/main/java/org/instituteatri/deep/controller/UserController.java

```diff
@@ -15,21 +24,124 @@
 @RequiredArgsConstructor
+@Tag(name = "User Controller", description = "Endpoints for user management")
 public class UserController {
 
     private final UserService userService;
 
+    @Operation(
+            method = "GET",
+            summary = "Get all users.",
+            description = "Returns a list of all users registered in the system, only with the admin token.")
+    @ApiResponses(value = {
+            @ApiResponse(
+                    responseCode = "200",
+                    description = "Success.",
+                    content = @Content(mediaType = "application/json",
+                            examples = @ExampleObject(
+                                    value = "{" +
+                                            "\"id\":\"string\"" +
+                                            ",\"name\":\"string\"," +
+                                            "\"email\":\"string\"}"
+                            ))),
+
+            @ApiResponse(responseCode = "403", description = "Forbidden.",
+                    content = @Content(mediaType = "application/json",
+                            examples = @ExampleObject(
+                                    value = "{\"message\":\"User isn't authorized.\"}"
+                            )))
+    })
     @GetMapping()
     public ResponseEntity<List<UserResponseDTO>> getAllUsers() {
         return userService.getAllUsers();
     }
 
+    @Operation(
+            method = "GET",
+            summary = "Get an user by ID",
+            description = "Returns the user with the specified ID. Only users with the ADMIN role can search by user.")
+    @ApiResponses(value = {
+            @ApiResponse(
+                    responseCode = "200",
+                    description = "Success.",
+                    content = @Content(mediaType = "application/json",
+                            examples = @ExampleObject(
+                                    value = "{" +
+                                            "\"id\":\"string\"" +
+                                            ",\"name\":\"string\"," +
+                                            "\"email\":\"string\"}"
+                            ))),
+
+            @ApiResponse(
+                    responseCode = "404",
+                    description = "Not found.",
+                    content = @Content(mediaType = "application/json",
+                            examples = @ExampleObject(
+                                    value = "{\"message\":\"Could not find user with id:661eff5e4af2c96e8a7dedc92\"}"
+                            ))),
+
+            @ApiResponse(
+                    responseCode = "403",
+                    description = "Forbidden.",
+                    content = @Content(mediaType = "application/json",
+                            examples = @ExampleObject(
+                                    value = "{\"message\":\"User isn't authorized.\"}"
+                            )))
+    })
```

Comment:
Test comment 2

Reply:
In that case, is it to remove these comments?